3_javascript/javascript1/week2/homework2ndweek/weather.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Weather</title>
+  </head>
+  <body>
+    <script>
+      function weather(temp) {
```

Comment:
you can be a bit more creative with the functions name.
the prop `temp` try to use `temperature` instead, the props should alaways be clear and meaningful.

Reply:
temp is clear for me

Reply:
I know, and it is also clear to me as well.
The function is 100% correct.
But imagine you have a file with too many functions, and every function has too many props, and you have many teammates that work with you, then you should try to make everything clear for everyone.